### PR TITLE
Add Kelli to team!

### PR DESCRIPTION
**Pull request by courier-new**
Added myself to the main team page, added my geo location, and filled in my name link from the Batchers page. 😄

### CODE REVIEW

**Review by LawnGnome — APPROVED** [review]
Welcome!

tenor

File: company/team/index.md [hunk 1]
```diff
@@ -856,3 +856,13 @@ Outside of work, he enjoys golfing, cheering on his favorite LA sports teams and
 - [rafal@sourcegraph.com](mailto:rafal@sourcegraph.com), [LinkedIn](https://www.linkedin.com/in/rleszcz/)
 - Raf lives with his family outside of Poznan (Poland), surrounded by forests, boars, deers, and other wildlife.
 He likes to travel, visit places off the beaten path, and trek in the mountains. Raf also spends bing chunk of his free time playing video games, both recent titles and retro ones. He has ten different gaming consoles, including NES, the first-generation Game Boy, and Commodore 64, all in working condition.
+
+
```
Comment (eseliger) [inline company/team/index.md:860]:
```suggestion

```
So sorry!

**Review by eseliger — APPROVED** [review]
Woho!

File: company/team/index.md (hunk 1 above)
Comment (courier-new) [reply company/team/index.md:860]: Oh no, where'd this line come from?? 😂 Sorry I'm used to a markdown linter haha.

**Comment by mrnugget** [comment]
Woop woop woop!

**Comment by chrispine** [comment]
yay!

**Pull request merged**